WordPress/Classes/ViewRelated/NUX/LoginSegueHandler.swift

```diff
@@ -10,3 +10,9 @@ extension LoginSegueHandler where Self: NUXAbstractViewController {
         performSegue(withIdentifier: identifier.rawValue, sender: sender)
     }
 }
+
+extension LoginSegueHandler where Self: NUXViewController {
```

Comment:
What do you think renaming this protocol & methods to not be Login specific? Since it's no longer just used for Login, it could be confusing.

Reply:
Ya, I think you're right.